corehq/apps/app_manager/static/app_manager/js/details/column.js

```diff
@@ -125,6 +162,15 @@ hqDefine("app_manager/js/details/column", function () {
             }
         }());
 
+        // TODO: use self.field.observableVal instead, and do something similar for header?
```

Comment:
@orangejenny Do you mean move `self.setTileContent();` to line 123 above?

Reply:
No, I was debating updating `setTileContent` to stop using the display property's header and instead use the `observableVal` (the case property name).